srcpkgs/skopeo/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'skopeo'
+pkgname=skopeo
+version=0.1.31
+revision=1
+build_style=go
+go_import_path="github.com/projectatomic/${pkgname}"
+go_package="${go_import_path}/cmd/${pkgname}"
+hostmakedepends="pkg-config"
+makedepends="device-mapper-devel go-md2man gpgme-devel libbtrfs-devel
```

Comment:
have go-md2man on hostmakedepends, otherwise you won't be able to run it when cross-compiling